packages/react-console/src/components/VncConsole/VncConsole.tsx

```diff
@@ -88,16 +88,18 @@ export const VncConsole: React.FunctionComponent<VncConsoleProps> = ({
   textSendShortcut,
   textCtrlAltDel
 }) => {
-  let rfb: any;
+  const rfb = React.useRef<any>();
   let novncStaticComponent: React.ReactNode;
   let novncElem: HTMLDivElement;
 
   const [status, setStatus] = React.useState(CONNECTING);
 
   const addEventListeners = () => {
-    rfb.addEventListener('connect', onConnected);
-    rfb.addEventListener('disconnect', onDisconnected);
-    rfb.addEventListener('securityfailure', onSecurityFailure);
+    if (rfb.current) {
+      rfb.current?.addEventListener('connect', onConnected);
+      rfb.current?.addEventListener('disconnect', onDisconnected);
```

Comment:
s/onDisconnected/_onDisconnected
s/onSecurityFailure/_onSecurityFailed


So that we update the status with the internal functions.

Reply:
Nice catch! Fixed! =)